src/sr.ml

```diff
@@ -69,6 +70,64 @@ let build_meta_type : int -> term = fun k ->
   let tk = _Meta mk (Array.map _Vari vs) in
   Bindlib.unbox (build_prod k tk)
 
+(** Translation from Xml-light
```

Comment:
The translation is not from Xml-light but from CPF. Add URL http://cl-informatik.uibk.ac.at/software/cpf/. xml is just the data structure used to represent the CPF file.

Reply:
It is probably better to put this function in a separate cpf.ml file.

Reply:
I am not sure if this is a good idea since what I am doing is not for general use. This is only used for translating CPF files generated by external completion tools.

Reply:
Just make this clear at the beginning of the file.